lib/rbac/acls.rb

```diff
@@ -0,0 +1,63 @@
+module RBAC
+  module ACLS
+    def new_acls(resource_id, permissions)
+      permissions.collect do |permission|
+        add_acl(permission, resource_id)
+      end
+    end
+
+    def remove_acls(acls, resource_id, permissions)
```

Comment:
Wondering if `remove` would work better than `remove_acls` ( and the other methods in this class ) since we know we're working on the `ACL` class already? ( similar to comment below in the `Roles` class.